index.html

```diff
@@ -3206,8 +3206,8 @@ <h3 id="lifetime-AudioNode">
             There are several types of references:
           </p>
           <ol>
-            <li>A <em>normal</em> JavaScript reference obeying normal garbage
-            collection rules.
+            <li>A <em>normal</em> reference obeying normal garbage collection
```

Comment:
Yes, I think spec often talk about "references" and not "javascript references" because the fact that we're using JavaScript is an implementation detail.

Quoting the [HTML spec][html] for example:

>This specification does not require support of any particular network protocol, style sheet language, scripting language, or any of the DOM specifications beyond those required in the list above. However, the language described by this specification is biased towards CSS as the styling language, JavaScript as the scripting language, and HTTP as the network protocol, and several features assume that those languages and protocols are in use.

[html]: https://html.spec.whatwg.org/#infrastructure